api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/IntensiveMonitoringCohortQueries.java

```diff
@@ -1446,21 +1446,21 @@ public CohortDefinition getMI15G() {
     map.put("856", hivMetadata.getHivViralLoadConcept().getConceptId());
 
     String query =
-            " SELECT p.patient_id "
-                    + " FROM   patient p "
-                    + "       INNER JOIN encounter ee "
-                    + "               ON ee.patient_id = p.patient_id "
-                    + "       INNER JOIN obs oo"
-                    + "               ON oo.encounter_id = ee.encounter_id "
-                    + " WHERE  p.voided = 0 "
-                    + "       AND ee.voided = 0 "
-                    + "       AND oo.voided = 0 "
-                    + "       AND ee.location_id = :location "
-                    + "       AND ee.encounter_type = ${6} "
-                    + "       AND oo.concept_id = ${856} "
-                    + "       AND oo.value_numeric >= 1000 "
-                    + "       AND ee.encounter_datetime <=  :endDate "
-                    + " GROUP BY p.patient_id                        ";
+        " SELECT p.patient_id "
+            + " FROM   patient p "
+            + "       INNER JOIN encounter ee "
+            + "               ON ee.patient_id = p.patient_id "
+            + "       INNER JOIN obs oo"
+            + "               ON oo.encounter_id = ee.encounter_id "
+            + " WHERE  p.voided = 0 "
+            + "       AND ee.voided = 0 "
+            + "       AND oo.voided = 0 "
+            + "       AND ee.location_id = :location "
+            + "       AND ee.encounter_type = ${6} "
+            + "       AND oo.concept_id = ${856} "
+            + "       AND oo.value_numeric >= 1000 "
```

Comment:
If you meant to achieve Viral load suppression, then it should be <=1000, is that the case or the requirements wanted it different?